src/components/Modals/FiatRamps/config.ts

```diff
@@ -112,4 +115,25 @@ export const supportedFiatRamps: SupportedFiatRamp = {
       }
     },
   },
+  MtPelerin: {
+    label: 'fiatRamps.mtPelerin',
+    logo: mtPelerinLogo,
+    isImplemented: true,
+    supportsBuy: true,
+    supportsSell: true,
+    getBuyAndSellList: async () => {
+      const mtPelerinAssets = await getMtPelerinAssets()
+      return [mtPelerinAssets, mtPelerinAssets]
+    },
+    onSubmit: (action: FiatRampAction, assetId: AssetId, address: string) => {
+      try {
+        const ticker = adapters.AssetIdToMtPelerinTicker(assetId)
+        if(!ticker) throw new Error('Asset not supported by MtPelerin')
+        const mtPelerinCheckoutUrl = getMtPelerinLink(action, ticker, address)
```

Comment:
Semantic and non functionally blocking but:
- use the `make` terminology, since we're not "get"ting anything from anywhere but making/building a link
- keep the same terminology on the left an right side of the assignment

```suggestion
        const mtPelerinCheckoutUrl = makeMtPelerinCheckoutUrl(action, ticker, address)
```